www/docs/client/links/httpBatchStreamLink.md

```diff
@@ -0,0 +1,50 @@
+---
+id: httpBatchStreamLink
+title: HTTP Batch Stream Link
+sidebar_label: HTTP Batch Stream Link
+slug: /links/httpBatchStreamLink
+---
+
+`unstable_httpBatchStreamLink` (experimental) is a [**terminating link**](./index.md#the-terminating-link) that batches an array of individual tRPC operations into a single HTTP request that's sent to a single tRPC procedure (equivalent to [`httpBatchLink`](./httpBatchLink.md)), but doesn't wait for all the responses of the batch to be ready and streams the responses as soon as any data is available.
+
+## Usage
+
+All usage and options are identical to [`httpBatchLink`](./httpBatchLink.md).
+
+## Streaming mode (experimental)
+
+> ⚠️ This link is experimental and may change in the future.
```

Comment:
Use the term `unstable` instead - https://trpc.io/docs/faq#unstable

The way I reason about things is that "experimental" is badly tested things that might be changed or dropped and unstable things are things that we sort-of committed to but might have issues

tl;dr: experimental ≈ alpha, unstable ≈ beta

I think this should be 'beta"